benchmarks/benchmark_serving.py

```diff
@@ -297,17 +297,41 @@ def sample_random_requests(
 async def get_request(
     input_requests: List[Tuple[str, int, int]],
     request_rate: float,
+    burstiness: float,
```

Comment:
```suggestion
    burstiness: float = 1.0,
```
To better align with the method documentation that this is optional